server.js

```diff
@@ -94,7 +95,7 @@ module.exports = function() {
         },
         validate: {
           payload: {
-            currency: Joi.string().min(3).max(3).required(),
+            currency: Joi.any().valid(currencyFor.stripe),
```

Comment:
I did a test and got the following response:

```
{ statusCode: 400,
  error: 'Bad Request',
  message: 'child "currency" fails because ["currency" is required]',
  validation: { source: 'payload', keys: [ 'currency' ] } }
```